iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/optional.inl

```diff
@@ -179,9 +194,9 @@ constexpr inline bool optional<T>::operator==(const optional<T>& rhs) const noex
 {
     return (!m_hasValue && !rhs.m_hasValue) || ((m_hasValue && rhs.m_hasValue) && (value() == rhs.value()));
 }
-
+// AXIVION Next Construct AutosarC++19_03-A0.1.4 : TODO
 template <typename T>
-constexpr inline bool optional<T>::operator==(const nullopt_t& rhs IOX_MAYBE_UNUSED) const noexcept
+constexpr inline bool optional<T>::operator==(const nullopt_t rhs IOX_MAYBE_UNUSED) const noexcept
```

Comment:
When we remove the variable name here we can get rid of `IOX_MAYBE_UNUSED` as well.